Scripts/README.MD

```diff
@@ -0,0 +1,6 @@
+# Scripts
+This folder contains scripts to compile AssemblyScript(AS), minify CSS and JS, and insert the import tags in it.  
+Those scripts produce two environement : 
+ - a developpement environments where the CSS and JS are not minifyed, the AS is comipiled in a laizy way to reduce compilation time and allow for fast trial and error.
```

Comment:
environments →environment
minifyed → minified
is comipiled → compilied
laizy → lazy